frontend/pages/hosts/details/cards/Policies/HostPoliciesTable/HostPoliciesTableConfig.tsx

```diff
@@ -83,19 +90,25 @@ const generatePolicyTableHeaders = (
         />
       ),
       disableSortBy: false,
-      sortType: "caseInsensitive",
-      accessor: "response",
+      sortType: "hostPolicyStatus",
+      accessor: "status",
       Cell: (cellProps) => {
-        if (cellProps.row.original.response === "") {
+        const {
+          row: {
+            original: { status },
+          },
+        } = cellProps;
+        if (status === null) {
           return <>{DEFAULT_EMPTY_CELL_VALUE}</>;
         }
-
-        const responseValue =
-          STATUS_CELL_VALUES[cellProps.row.original.response];
+        const [
+          indicatorStatus,
+          displayText,
+        ] = POLICY_STATUS_TO_INDICATOR_PARAMS[status];
```

Comment:
is it possible for `status` to be `null` here?  I see that's a possible output from `getPolicyStatus()`

Reply:
yessir, handled [here](https://github.com/fleetdm/fleet/pull/28658/files/fd104db34cf47b613f72fd30fa89716c0b2dd9e2#diff-f1fa796dd5eb53e135a4132771bbcf85c168a5c5bcdf9c3d9039e83670ec1cb6R101)

Reply:
ahh 👍 👓